controlsfx/src/main/java/impl/org/controlsfx/skin/HyperlinkLabelSkin.java

```diff
@@ -70,52 +70,59 @@ public class HyperlinkLabelSkin extends SkinBase<HyperlinkLabel> {
             }
         }
     };
-    
-    
 
     /***************************************************************************
      * 
      * Constructors
      * 
      **************************************************************************/
-    
+
     public HyperlinkLabelSkin(HyperlinkLabel control) {
         super(control);
-        
-        this.textFlow = new TextFlow();
+
+        textFlow = new TextFlow();
+        hyperlink = new Hyperlink();
+        hyperlink.setPadding(new Insets(0, 0, 0, 0));
+        hyperlink.setOnAction(eventHandler);
+        hyperlink.focusTraversableProperty().bind(control.focusTraversableProperty());
+        control.focusedProperty().addListener((o, ov, nv) -> {
```

Comment:
Just a minor concern whether this should be inside a WeakListener or not. I had so many memory leaks in applications because of listener added directly. 
Don't know if this one can cause a memory leak or not

Reply:
Valid point. I have overridden `dispose` method to unbind bindings and remove listeners.

Reply:
I now use `SkinBase#registerChangeListener` which is called and gc'd during dispose automatically. Another round of review is appreciated.